src/main/java/Operator.java

```diff
@@ -0,0 +1,46 @@
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.function.BiFunction;
+
+public enum Operator {
+    PLUS("+", (a, b) -> {
+        return a + b;
+    }),
+    MINUS("-", (a, b) -> {
+        return a - b;
+    }),
+    DIVE("/", (a, b) -> {
+        return a / b;
+    }),
+    MULTIPLY("*", (a, b) -> {
+        return a * b;
+    });
+
+    private String value;
+    private BiFunction<Integer, Integer, Integer> calc;
+
+    Operator(String value, BiFunction<Integer, Integer, Integer> calc) {
+        this.value = value;
+        this.calc = calc;
+    }
+
+    public static Operator findOperator(String input) {
+        Optional<Operator> op = Arrays.stream(values())
+                .filter(operator -> isEqual(operator, input))
+                .findAny();
+
+        if (op.isEmpty()) {
+            throw new IllegalArgumentException("올바르지 않은 수식입니다.");
+        }
+
+        return op.get();
```

Comment:
```suggestion
        return Arrays.stream(values())
                .filter(operator -> isEqual(operator, input))
                .findAny()
                .orElseThrow(() -> new IllegalArgumentException("올바르지 않은 수식입니다."));
```
조금 더 간단하게 표현할 수 있어요!